crates/observe/src/tracing.rs

```diff
@@ -2,7 +2,13 @@ use {
     std::{panic::PanicInfo, sync::Once},
     time::macros::format_description,
     tracing::level_filters::LevelFilter,
-    tracing_subscriber::fmt::{time::UtcTime, writer::MakeWriterExt as _},
+    tracing_subscriber::{
+        fmt::{time::UtcTime, writer::MakeWriterExt as _},
+        prelude::__tracing_subscriber_SubscriberExt,
```

Comment:
Typically, `__*` names are used for "internal" types that need to be public because of compiler limitations/reasons.

Since `prelude` modules are intended to be used with `*`, I would change this to:

```suggestion
        prelude::*,
```